tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/FindReplaceDialogTest.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2000, 2016 IBM Corporation and others.
+ * Copyright (c) 2024 Vector Informatik GmbH and others.
```

Comment:
Please don't change the existing copyright information (in particular do not replace the company name).

Reply:
This was not an intended change and stems from a blind Copy-Paste